Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -112,12 +88,20 @@ void PixelShaderManager::SetConstants()
 		dirty = true;
 		s_bViewPortChanged = false;
 	}
+
+	if (s_bEFBScaleChanged)
+	{
+		constants.efbscale[0] = 1.0f / float(Renderer::EFBToScaledXf(1));
+		constants.efbscale[1] = 1.0f / float(Renderer::EFBToScaledYf(1));
```

Comment:
Yes, that's why we don't need to check for s_bEFBScaleChanged thousend times per frame.